test/openscap_parser/test_result_file_test.rb

```diff
@@ -416,6 +416,294 @@ def setup
+                           'xccdf_org.ssgproject.content_value_var_polyinstantiation_enabled' => 'on',
+                           'xccdf_org.ssgproject.content_value_sysctl_net_ipv4_conf_all_accept_redirects_value' => 'disabled',
+                           'xccdf_org.ssgproject.content_value_sysctl_net_ipv4_conf_default_accept_redirects_value' => 'disabled',
+                           'xccdf_org.ssgproject.content_value_var_selinux_policy_name' => 'targeted',
+                           'xccdf_org.ssgproject.content_value_var_accounts_user_umask' => '077',
+                           'xccdf_org.ssgproject.content_value_var_accounts_tmout' => '10_min',
+                           'xccdf_org.ssgproject.content_value_sshd_idle_timeout_value' => '10_minutes',
+                           'xccdf_org.ssgproject.content_value_var_selinux_state' => 'enforcing' },
+                         report.benchmark.profiles.first.refined_values)
+            # rubocop:enable Layout/LineLength
+          end
+        end
+
+        context 'groups' do
+          test "#{report_type}_group_id" do
+            assert_match(/^xccdf_org.ssgproject.content_group_system/,
+                         report.benchmark.groups.first.id)
+          end
+
+          test "#{report_type}_group_no_conflicts" do
+            assert_equal([], report.benchmark.groups.first.conflicts)
+          end
+
+          test "#{report_type}_group_no_requires" do
+            assert_equal([], report.benchmark.groups[1].requires)
+          end
+
+          test "#{report_type}_group_description" do
+            assert_match(/^Contains rules that check correct system settings./,
+                         report.benchmark.groups.first.description)
+          end
+
+          test "#{report_type}_group_parent_id_benchmark" do
+            assert_match(/^xccdf_org.ssgproject.content_benchmark_RHEL-9/,
+                         report.benchmark.groups.first.parent_id)
+          end
+
+          test "#{report_type}_group_parent_id_group" do
+            assert_match(/^xccdf_org.ssgproject.content_group_system/,
+                         report.benchmark.groups[1].parent_id)
+          end
+
+          test "#{report_type}_group_parent_type_with_benchmark_parent" do
+            assert_match(/^Benchmark/,
+                         report.benchmark.groups.first.parent_type)
+          end
+
+          test "#{report_type}_group_parent_type_with_group_parent" do
+            assert_match(/^Group/,
+                         report.benchmark.groups[1].parent_type)
+          end
+        end
+
+        context 'rules' do
+          test "#{report_type}_list all rules" do
+            arbitrary_rules = [
+              # rubocop:disable Metrics/LineLength
+              'xccdf_org.ssgproject.content_rule_bios_enable_execution_restrictions',
+              'xccdf_org.ssgproject.content_rule_selinux_all_devicefiles_labeled'
```

Comment:
Instead of disabling the LineLength rubocop check, you could put 'xccdf_org.ssgproject.content_rule_bios_enable_execution_restrictions' and 'xccdf_org.ssgproject.content_rule_selinux_all_devicefiles_labeled' on separate lines. We usually try to avoid disabling the rubocop checks if possible.